packages/block-library/src/post-author/edit.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { sprintf, __ } from '@wordpress/i18n';
+
+function PostAuthorDisplay() {
+	const [ authorId ] = useEntityProp( 'postType', 'post', 'author' );
+	const author = useSelect(
+		( select ) => select( 'core' ).getEntityRecord( 'root', 'user', authorId ),
+		[ authorId ]
+	);
+	return author ? <address>{ sprintf( __( 'By %s' ), author.name ) }</address> : null;
```

Comment:
In slow connection, there's a jumpiness happening when loading the author. This should be solved with design somehow.

Reply:
We need loading block placeholders for these "async blocks."